libsignal-service/src/push_service.rs

```diff
@@ -205,28 +229,24 @@ pub trait PushService {
     type WebSocket: WebSocketService;
     type ByteStream: futures::io::AsyncRead + Unpin;
 
-    async fn get<T>(&mut self, path: &str) -> Result<T, ServiceError>
+    async fn get<T>(&self, path: &str) -> Result<T, ServiceError>
```

Comment:
Actually, for the trait, I think it's still useful to use `&mut`, because I'm not quite sure that all http backends will have internal mutability for writing to a socket. What do you think?

Reply:
`hyper` doesn't need interior mutability as well, so I don't think it's a concern (see: https://docs.rs/hyper/0.13.8/hyper/client/struct.Client.html#method.get).